docs: Documentation update for OpenTelemetry (#915)
googleapis/python-logging: docs/auto-trace-span-extraction.rst

Automatic Trace/Span ID Extraction
==================================

The Google Cloud Logging library can automatically populate `LogEntry fields <https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry>`_

> `trace`, `span_id`, and`trace_sampled` via OpenTelemetry integration, or extracting header information from an HTTP request.

Reviewer: Somehow and`trace_sampled` is not populated correctly in the screenshot.

OpenTelemetry Integration
-------------------------

If you have the OpenTelemetry SDK package installed and are logging from within an active OpenTelemetry span, that log entry will automatically
have the `trace`, `span_id`, and `trace_sampled` fields populated with metadata from that span. More information about OpenTelemetry can be found 
`here <https://opentelemetry.io/docs/languages/python/>`_.

HTTP headers
------------

> Another possible method of automatic `trace`/ `span_id` is via extraction from HTTP headers. This feature requires a :doc:`supported Python web framework </web-framework-integration>`.

Reviewer: Do we want to mention that this is prioritized after OpenTelemetry?

Reviewer 2: I think that's probably a good idea.

Automatic Trace/Span ID Extraction
==================================

.. note::
    All `LogEntry fields`_ populated :ref:`manually<Manual-Metadata>` will override those populated via methods referred to in this
    section.

The Google Cloud Logging library can automatically populate `LogEntry fields`_
`trace`, `span_id`, and `trace_sampled` via OpenTelemetry integration, or extracting header information from an HTTP request.

OpenTelemetry Integration
-------------------------

> If you have the OpenTelemetry SDK package installed and are logging from within an active OpenTelemetry span, that log entry will automatically

Reviewer 3: nit the SDK is not technically necessary here. Does this suggestion make sense to you? I think it's a little more precise
```suggestion
If you are using OpenTelemetry and there is an active span in the OpenTelemetry Context, that log entry will automatically
```

> have the `trace`, `span_id`, and `trace_sampled` fields populated with metadata from that span. More information about OpenTelemetry can be found 

Reviewer 3: nit

```suggestion
have the `trace`, `span_id`, and `trace_sampled` fields populated from that span. More information about OpenTelemetry can be found 
```
